### PR TITLE
release: v3.4.0-rc.2

### PR DESCRIPTION
**Pull request by PavelMor25**
## Approach
update version release in package and package-lock
update method _updateTestControllerQueues in src\runner\browser-job.ts

## Pre-Merge TODO
- [x] Write tests for your proposed changes
- [x] Make sure that existing tests do not fail

### CODE REVIEW

**Review by AlexKamaev — APPROVED** [review]

File: src/runner/browser-job.ts [hunk 1]
```diff
@@ -214,13 +214,20 @@ export default class BrowserJob extends AsyncEventEmitter {
         return this._testRunControllerQueue;
     }
 
+    private _getlastIndexFixture ( fixtureId: string ): number {
+        const lastIndexFixture = this._testRunControllerQueue.findIndex(el => el.test.fixture?.id !== fixtureId);
+
+        if (!~lastIndexFixture)
```
Comment (aleks-pro) [inline src/runner/browser-job.ts:220]: The following statement is much easier to read:
```suggestion
        if (lastIndexFixture === -1)
```

File: src/runner/browser-job.ts [hunk 2]
```diff
@@ -214,13 +214,20 @@ export default class BrowserJob extends AsyncEventEmitter {
         return this._testRunControllerQueue;
     }
 
+    private _getlastIndexFixture ( fixtureId: string ): number {
+        const lastIndexFixture = this._testRunControllerQueue.findIndex(el => el.test.fixture?.id !== fixtureId);
```
Comment (aleks-pro) [inline src/runner/browser-job.ts:218]: What do you mean by `lastIndexFixture`?

Comment (PavelMor25) [reply src/runner/browser-job.ts:218]: Last index current fixture with disable concurrency

Comment (PavelMor25) [reply src/runner/browser-job.ts]: I rename `lastIndexFixture` and method ` _getlastIndexFixture` to  `nextFixtureFirstTestIndex` and `_getNextFixtureFirstTestIndex`. Last names was incorrect.

Method returns next fixture's test's first index.

**Review by aleks-pro — APPROVED** [review]

**Pull request merged**